queue/ipfs.go

```diff
@@ -111,7 +112,7 @@ func (qm *QueueManager) ProcessIPFSKeyCreation(msgs <-chan amqp.Delivery, db *go
 			d.Ack(false)
 			continue
 		}
-		err = userManager.AddIPFSKeyForUser(key.UserName, key.Name, id.Pretty())
+		err = userManager.AddIPFSKeyForUser(key.UserName, keyName, id.Pretty())
```

Comment:
as per https://github.com/RTradeLtd/Temporal/wiki/Code-Formatting-Guidelines, this should be inlined right?

Reply:
yes it should :ok_hand:  good catch